tests/test_data.py

```diff
@@ -162,3 +146,37 @@
     "triggersIds": [],
     "alertId": 304255
 }
+
+alert_D = {
+    "id": testId,
+    "createdById": 101997,
+    "createdDate": 1459857033871,
+    "modifiedById": 101997,
+    "modifiedDate": 1459857033871,
+    "name": alertName,
+    "expression": "-1d:hdara:test:sum",
+    "cronEntry": "*/15 * * * *",
+    "enabled": False,
+    "missingDataNotificationEnabled": False,
+    "notificationsIds": [],
+    "triggersIds": [],
+    "ownerName": userName
+}
+
+alert_all_info_D = {
+    "id": testId,
+    "createdById": 101997,
+    "createdDate": 1459857033871,
+    "modifiedById": 101997,
+    "modifiedDate": 1459857033871,
+    "name": alertName,
+    "expression": "-1d:hdara:test:sum",
+    "cronEntry": "*/15 * * * *",
+    "enabled": False,
+    "missingDataNotificationEnabled": False,
+    "notificationsIds": [testId, testId, testId],
+    "triggersIds": [testId, testId],
+    "triggers": [trigger_D, trigger_D],
+    "notifications": [notification_D, notification_D, notification_D],
```

Comment:
I don't see any use of the repetition here, at least for the sake of the new API that you are testing. We could test this in the model related methods, but we don't have any right now.

Reply:
Ok, added, please take a look